packages/@magic-ext/oauth2/src/index.ts

```diff
@@ -44,7 +44,10 @@ export class OAuthExtension extends Extension.Internal<'oauth2'> {
       }
 
       if (successResult?.oauthAuthoriationURI) {
-        window.location.href = successResult.oauthAuthoriationURI;
+        window.location.href = successResult.useMagicServerCallback
+          ? // @ts-ignore - this.sdk.endpoint is marked protected but we need to access it.
```

Comment:
how about adding a public getter function for `endpoint` so we don't have to do this?

Reply:
That's a good idea

Reply:
Yeah, I was thinking the same thing since this is protected. Didn't want to muddy up this PR so  perhaps we can make a separate PR for this?